src/checker.cpp

```diff
@@ -6302,7 +6302,7 @@ gb_internal void check_deferred_procedures(Checker *c) {
 					gbString s = type_to_string(tsrc);
 					gbString d = type_to_string(dst_params);
 					error(src->token, "Deferred procedure '%.*s' parameters do not match the results of initial procedure '%.*s':\n\t(%s) =/= (%s)",
-					      LIT(src->token.string), LIT(dst->token.string),
+					      LIT(dst->token.string), LIT(src->token.string),
 					      s, d
```

Comment:
I would also switch `s, d` in these cases, so it also flips around the initial and deferred input types.
As written it would result in:
```
W:/Odin/bug/bug.odin(10:1) Error: Deferred procedure 'proc1' parameters do not match the inputs of initial procedure 'proc0': 
	(int) =/= (f32) 
	proc0 :: proc(param: int) { 
	^ 
```
When you'd want:
```
W:/Odin/bug/bug.odin(10:1) Error: Deferred procedure 'proc1' parameters do not match the inputs of initial procedure 'proc0': 
	(f32) =/= (int)
	proc0 :: proc(param: int) { 
	^ 
```

Reply:
done! good catch